my-app/src/helpers/services.ts

```diff
@@ -0,0 +1,45 @@
+import axios from "../axios";
+import {BlogPostType} from "../pages/homePage/BlogHomeContext";
+
+type InputType = {
+    title: string
+    text: string
+    imageUrl: string
+}
+type RegisterInput = {
+    email: string,
+    password: string,
+    fullName: string,
+}
+type LoginInput = {
+    email: string,
+    password: string,
+}
+export const services = {
+    blog: {
+        list: async () => {
+            const response = axios.get('/posts').then(res => res.data)

```

Comment:
not necessarily bad, but I thought you would make use of your API scheme. If you already have it, you should not call api directly throuhg axios. Generate functions from schema (automatically) that will call BE. 

try this https://www.npmjs.com/package/swagger-axios-codegen

or this https://www.npmjs.com/package/swagger-react-generator

The point is that when you already defined `GET /posts` in schema. You will run one command on FE where your schema will be an input and the script will generate a set of functions for you. The functions will be in one autogenerated file and will be named smth like `getPosts()` 

then you will just import this functions in your code and call them directly